components/var_dumper/introduction.rst

```diff
@@ -62,6 +62,17 @@ current PHP SAPI:
     You'll also learn how to change the format or redirect the output to
     wherever you want.
 
+.. tip::
+
+    In order to have the ``dump()`` function always available, you can install
+    it globally in your dev environment:
```

Comment:
I do agree with you, but I also don't like the verbose thing you proposed. The docs are always a fight between making something easy to understand and keeping something as short as possible (especially when it's about simple tips like this).

What do you think about replacing "environment" with "machine" or "box"?